frontend/src/tabs/nfts/NFTs.js

```diff
@@ -18,25 +30,33 @@ const NFTsMainSection = (props) => {
           Mint
         </div>
       </div>
-      <div className='NFTs__container'>
-        {props.nftsCollection.map((nft, index) => {
-          return (
-            <NFTItem
-              key={index}
-              tokenId={nft.tokenId}
-              position={nft.position}
-              image={imageURL + 'nft-' + nft.tokenId + '.png'}
-              width={nft.width}
-              height={nft.height}
-              blockNumber={nft.blockNumber}
-              likes={nft.likes}
-              liked={nft.liked}
-              minter={nft.minter}
-              queryAddress={props.queryAddress}
-            />
-          );
-        })}
-      </div>
+      <>
+        <div className='NFTs__container'>
+          {props.nftsCollection.map((nft, index) => {
+            return (
+              <NFTItem
+                key={index}
+                tokenId={nft.tokenId}
+                position={nft.position}
+                image={imageURL + 'nft-' + nft.tokenId + '.png'}
+                width={nft.width}
+                height={nft.height}
+                blockNumber={nft.blockNumber}
+                likes={Math.floor(Math.random() * 1000)}
+                liked={Math.random() > 0.5}
+                minter={nft.minter}
+              />
+            );
+          })}
+        </div>
+        <PaginationView
+          totalPages={mockNftData.totalPages}
+          pageLength={mockNftData.pageLength}
```

Comment:
I don't think mock data is needed here since the page length is defined in this file?